docusaurus.config.ts

```diff
@@ -277,6 +274,13 @@ const config: Config = {
       theme: prismThemes.github,
       darkTheme: prismThemes.dracula,
     },
+    // Change website analytics from Google Analytics to Matomo
+    matomo: {
+      matomoUrl: 'https://analytics.apache.org/',
+      siteId: 'ID',
```

Comment:
Now have asked for one Site ID from the ASF Privacy team. Once got it, will update this field.